### ISSUE TITLE
Javascript error thrown on page with no CMP

### ISSUE THREAD

**Issue by mgscox** [cavi-au/Consent-O-Matic#78]
Seen whilst running a local development server (which does not present a CMP popup)

```
Unknown action type: ifallowall
createAction @ Action.js:18
(anonymous) @ CMP.js:15
CMP @ CMP.js:13
(anonymous) @ ConsentEngine.js:15
ConsentEngine @ ConsentEngine.js:13
(anonymous) @ contentScript.js:23
Promise.then (async)
(anonymous) @ contentScript.js:15
Promise.then (async)
(anonymous) @ contentScript.js:14
Promise.then (async)
(anonymous) @ contentScript.js:7
```
Browser: Google Chrome Version 102.0.5005.115 (Official Build) (x86_64)
OS: MacOS Monteray 12.4

Relevant object dumps from CMP.js

**CMP**
```
{
    "name": "Facebook",
    "detectors": [
        {
            "config": {
                "presentMatcher": [
                    {
                        "type": "css",
                        "target": {
                            "selector": "[data-testid=\"cookie-policy-manage-dialog\"]"
                        }
                    }
                ],
                "showingMatcher": [
                    {
                        "type": "css",
                        "target": {
                            "selector": "[data-testid=\"cookie-policy-manage-dialog\"]",
                            "displayFilter": true
                        }
                    }
                ]
            },
            "presentMatchers": [
                {
                    "config": {
                        "type": "css",
                        "target": {
                            "selector": "[data-testid=\"cookie-policy-manage-dialog\"]"
                        }
                    }
                }
            ],
            "showingMatchers": [
                {
                    "config": {
                        "type": "css",
                        "target": {
                            "selector": "[data-testid=\"cookie-policy-manage-dialog\"]",
                            "displayFilter": true
                        }
                    }
                }
            ]
        }
    ],
    "methods": {}
}
```
**methodConfig**
```
{
    "action": {
        "type": "ifallowall",
        "trueAction": {
            "type": "click",
            "target": {
                "selector": "[data-cookiebanner=\"accept_only_essential_button\"]"
            }
        },
        "falseAction": {
            "type": "click",
            "target": {
                "selector": "[data-cookiebanner=\"accept_button\"]"
            }
        }
    },
    "name": "DO_CONSENT"
}
```
```
name: "Facebook"
```

**Comment by dkgof** [comment]
It is because we have added some new actions, that wont be available until release of v1.0.4 of the extension.

But the rules start updating at once.

The error should only mean, that the CMP using the error, is ignored until the action is available.

**Comment by BojanKogoj** [comment]
Any way to make it silently fail for future changes?

**Comment by dkgof** [comment]
Yes!

The new version, will not spit out an error should the same occur again. Hopefully the new version is out soon.

**Comment by svnhub** [comment]
This fix is included in v1.0.8 which is being released
